app/controllers/api/v1/seances_controller.rb

```diff
@@ -6,14 +6,14 @@ def index
   end
 
   def show
-    @seance = Seances::UseCases::Show.new.call(id: params[:id])
+    @seance = Seances::Repository.new.find(params[:id])
     render json: Seances::Representers::OneSeance.new(@seance).basic
   end
 
   def create
-    @cinema_hall = CinemaHall.find(params[:cinema_hall_id])
-    @seance = @cinema_hall.seances.create(seance_params)
-    @seance = Seances::UseCases::Create.new.call(params: seance_params)
+    @cinema_hall = CinemaHalls::UseCases::Show.new.call(id: params[:cinema_hall_id])
+    @seance = Seances::UseCases::Create.new.call(params: seance_params.merge(cinema_hall_id: params[:cinema_hall_id]))
```

Comment:
@seance = Seances::UseCases::Create.new.call(params: seance_params.merge(cinema_hall_id: params[:cinema_hall_id]))
  @seance = Seances::UseCases::Create.new.call(params: seance_params.merge(movie_id: params[:movie_id]))
 I need to connect those 2 create method into 1, and I don't know how. @w0st

Reply:
Let's discuss it during the call, we can simplify it.